tree/ntuple/v7/test/ntuple_types.cxx

```diff
@@ -529,15 +529,14 @@ TEST(RNTuple, StdMap)
       auto map_field2 = model->MakeField<std::map<int, std::vector<CustomStruct>>>({"myMap2"});
 
       auto myMap3 = RFieldBase::Create("myMap3", "std::map<char, std::string>").Unwrap();
-      auto myMap4 = RFieldBase::Create("myMap4", "std::map<float, std::map<char, std::int32_t>>").Unwrap();
```

Comment:
Why are we removing the test on `std:int32_t`?

Reply:
I changed it to `std::map<int, std::map<char, float>>` to test whether `int` gets properly normalized (this was not the case before this change).

Reply:
I think we need to keep the `std::int32_t` to test typedefs and simply add the new one.

Reply:
Let me offer a two-birds-with-one-stone-solution :-)

Reply:
If the extra run-time is not large, there is great advantage in term of a debug-ability to have the test grow in complexity one step at a time.